rust/Dockerfile

```diff
@@ -8,6 +8,7 @@ COPY lib/ /opt/zingo/rust/lib/
 COPY Cargo.lock /opt/zingo/rust/Cargo.lock
 COPY Cargo.toml /opt/zingo/rust/Cargo.toml
 COPY ios/ /opt/zingo/rust/ios/
+COPY test_utils/ /opt/zingo/rust/test_utils/
```

Comment:
If this becomes too slow, we can consider mounting volumes from the build context, or publishing `test_utils` as a crate and making it a dependency.

Reply:
sure, copying the files is a very quick step in the build so unlikely to botteneck here but open to optimisations later down the line